src/util/storageOptions.tsx

```diff
@@ -69,8 +71,9 @@ export const driversWithFilesystemSupport = [
   lvmDriver,
   cephDriver,
   pureStorage,
+  cephObject,
 ];
 
 export const isRemoteStorage = (driver: string) => {
-  return [cephDriver, cephFSDriver, powerFlex].includes(driver);
+  return [cephDriver, cephFSDriver, powerFlex, cephObject].includes(driver);
```

Comment:
```suggestion
  return [cephDriver, cephFSDriver, cephObject, powerFlex].includes(driver);
```
Best keep this sorted alphabetically.